### PR TITLE
add french translation for the module translation

### PR DESCRIPTION
**Pull request by Vilsafur**

### CODE REVIEW

File: Resources/lang/translation/fr/translations.php [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    ],
+    'messages' => [
+    ],
+    'validation' => [
+    ],
+    'Clear translation cache' => 'Nettoyer le cache des traductions',
+    'Export' => 'Exporter',
+    'Import' => 'Importer',
+    'Import csv translations file' => 'Importer un fichier de traductions',
+    'only for developers' => 'N&eacute;cessitera un gestion git !',
```
Comment (nWidart) [inline Resources/lang/translation/fr/translations.php]: You can use the usual UTF8 characters. Meaning é, è etc. are all ok. Use a single quote by escaping it `\'`

File: Resources/lang/translation/fr/translations.php [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    'Translations exported' => 'Traductions exportées.',
+    'Translations imported' => 'Traductions importées.',
+    'select CSV file' => 'Selectionner un fichier CSV ...',
+    'csv only allowed' => 'Charger uniquement un fichier CSV, s\'il vous plait.',
+    'time' => 'Temps',
+    'author' => 'Auteur',
+    'history' => 'Historique',
+    'No Revisions yet' => 'Pas encore de révisions',
+    'event' => 'Évenement',
+    'created' => 'Creéé le',
```
Comment (nWidart) [inline Resources/lang/translation/fr/translations.php]: Typo on Crée. :)

**Comment by nWidart** [comment]
Can you please squash those 3 commits please ( `git rebase -i HEAD~3`) 3, being the number of previous commits to squash

**Comment by joshbrw** [comment]
Just to add to the above, a nice squashing guide here for if you haven't done it before: http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html

**Comment by Houbsi** [comment]
'csv only allowed' => 'Charger uniquement un fichier CSV, s\'il vous plait.',

should be ‘Uniquement des fichiers CSV sont permis.'

**Comment by Houbsi** [comment]
'only for developers' => 'Nécessitera un gestion git !',
should be 'seulement pour les développeurs'

**Comment by Houbsi** [comment]
+    'Translations exported' => 'Traductions exportées.',
 +    'Translations imported' => 'Traductions importées.',

both without "es"

**Comment by joshbrw** [comment]
Tagged in 1.10.0: https://github.com/AsgardCms/Translation/releases/tag/1.10.0

**Pull request closed without merging**